pysam/libcbcf.pyx

```diff
@@ -2257,7 +2257,7 @@ cdef class VariantRecordFilter(object):
             index = key
 
             if index < 0 or index >= n:
-                raise IndexError('invalid filter index')
+                raise IndexError('Invalid filter index.')
```

Comment:
More than 95% of the error strings in both the Python stdlib and pysam don't have a trailing fullstop, so that's the convention I'd like to follow. I'm on board with capitalising these sentences, but let's leave them without a final period/fullstop.